index.html

```diff
@@ -0,0 +1,201 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- ICONTSCOUNTS LINK -->
+    <link rel="stylesheet" href="https://unicons.iconscout.com/release/v2.1.6/css/unicons.css">
+     <!-- GOOGLE FONTS LINKS -->
+     <link rel="preconnect" href="https://fonts.googleapis.com">
+     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+     <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;500;600;700&display=swap" rel="stylesheet">
+     <link rel="stylesheet" href="style.css">
+    <title>Capstone</title>
+</head>
+<body>
+<!-- NAVBAR STARTS -->
+
+<div class="nav-container-bg">
+  <nav class="top-navbar">
+    <ul>
+      <li><img src="imgs/facebook.png" alt="facebook-image"></li>
+      <li><img src="imgs/twitter.png" alt="twitter-image"></li>
+      <li>logout</li>
+      <li>Mypage</li>
+      <li>Korean</li>
+    </ul>
+  </nav>
+<nav>
+  <div class="container nav-container">
+    <div class="hamburger-btn">
+      <span class="bar"></span>
+      <span class="bar"></span>
+      <span class="bar"></span>
+    </div>
+    <div class="menu-js">
+      
+      <button id="close-btn" type="button" class="close-btn x">
+        <span>&times;</span>
+      </button>
+    <div class="image-nav">
+      <a href="index.html"> <img class="progarm-logo" src="imgs/logo-project.png" alt=""></a>
+      <ul class="nav-menu">
+          <li ><a class="link about"  href="#">About</a></li>
```

Comment:
- [ ] Kindly edit this link to redirect the user to the "about" page.
- [ ] Please add a way to navigate from the "about" page to the "home" page on the mobile view. (You can add a navlink in the mobile menu that navigates to the home page)

- [ ] Kindly adjust the size of the mobile menu to take 100% of the screen's height, to make your project more responsive. 
![image](https://user-images.githubusercontent.com/62290545/170580457-0927c8ee-411b-456c-8ec1-aba11c5c46c5.png)